mlflow/keras/autologging.py

```diff
@@ -205,16 +209,19 @@ def _patched_inference(original, inst, *args, **kwargs):
             unlogged_params.append("batch_size")
 
         log_fn_args_as_params(original, [], kwargs, unlogged_params)
+        model_id = None
+        if log_models:
+            model_id = mlflow.initialize_logged_model("model").model_id
```

Comment:
Should we specify `name="model"`? It may not be convenient for users if many keras models are logged by autologging with the same name.

Reply:
That's the current behavior though, so I don't want to break it in this PR. If we auto-generate model names, would it make any difference though, since users don't know the model name anyways 🤔